src/molecules/Avatar/Avatar.jsx

```diff
@@ -1,14 +1,49 @@
-import React from 'react';
-import { AvatarImg } from './styled';
-import { ButtonAtom } from '../../atoms';
-import Icon from '../../assets/img/profile_icon.png';
+import React, { useState, useContext } from 'react';
+import { Link, useHistory } from 'react-router-dom';
+import { AvatarImg, RelativePos } from './styled';
+import { ButtonAtom, MenuAtom, TextAtom } from '../../atoms';
+import { GlobalContext } from '../../states/GlobalContext';
+import PropTypes from 'prop-types';
 
 function Avatar({ className }) {
+  let [hide, setHide] = useState(true);
+  const [state, dispatch] = useContext(GlobalContext);
+  const history = useHistory();
+  const defaultImg = require('../../assets/img/profile_icon.png');
+  
+
+  function clickHandler() {
+    setHide(!hide)
+  }
+
+  function logout() {
+    dispatch({ type: 'logout' });
+    history.push('/login');
+  }
+
   return (
-    <ButtonAtom round padding="0.5rem" className={className}>
-      <AvatarImg src={Icon} />
-    </ButtonAtom>
+      <RelativePos onClick={clickHandler} onKeyDown={clickHandler}>
+        <ButtonAtom round padding="0.5rem" className={className}>
+          <AvatarImg src={state.session ? state.session.avatarUrl : defaultImg} alt="avatar" />
+        </ButtonAtom>
+        <MenuAtom hide={hide} siblingHeight="66px" alignRight>
+          { !state.session ? (
+            <Link to="/login">
+              <TextAtom type="label">Iniciar sesión</TextAtom>
+            </Link>
+          ) : (<div onClick={logout} onKeyDown={logout}><TextAtom type="label">Cerrar sesión</TextAtom></div>) 
```

Comment:
Was there any particular reason on why you chose `div` over your `Button` component?. If you want to keep it this way, the `div` should have the `role="button"` as it is for accessibility support. And also you won't get warn about it.